DNN Platform/DotNetNuke.Web.Mvc/DnnMvcControllerFactory.cs

```diff
@@ -0,0 +1,16 @@
+﻿using DotNetNuke.Common;
+using System;
+using System.Web.Mvc;
+using System.Web.Routing;
+
+namespace DotNetNuke.Web.Mvc
+{
+    class DnnMvcControllerFactory : DefaultControllerFactory, IControllerFactory
```

Comment:
This class is missing an access modifier

Reply:
Good catch!

Reply:
I added XML comments to this class as wel